CHANGES

```diff
@@ -1,3 +1,14 @@
+Release 2.3.0 (January 16, 2020)
+===============================
+
+* Fix formatting for pyupgrade
```

Comment:
Can we sort / group it in some way so that minor issues (formatting changes, minor documentation updates) do not cover the expected patches (fixing a bug in the UUID listed at the very end of the list)?

Reply:
moved "Fix compatibility with UUIDField primary keys" up